workshop/exercise-8/README.md

```diff
@@ -1,4 +1,4 @@
-# Exercise 8 - Policy Enforcement 
+# Exercise 8 - Policy Enforcement
 
 Backend systems such as access control systems, telemetry capturing systems, quota enforcement systems, billing systems, and so forth, traditionally directly integrate with Services, creating a hard coupling and baking-in specific semantics and usage options.
 
```

Comment:
I found the intro to this exercise to be difficult to understand. Maybe break it up into a couple of sentences? Maybe provide a bit more context for the signficance of some of this.

For example, why is this important: ` Its design moves policy decisions out of the app layer and into configuration instead, under operator control.`

Reply:
@szihai Please review the above issue, raised by a tester.